### ISSUE TITLE
ipython@5: inreplace failed

### ISSUE THREAD

**Issue by Char-Aznable** [Linuxbrew/homebrew-core#5703]
# Please fill out the issue checklist below, and provide all of the requested information.

# Please always follow these steps:
- [x] Confirmed this is a problem with `brew install`ing one, specific formula and not every time you run `brew`? If it's a general `brew` problem please file this issue at https://github.com/Linuxbrew/brew/issues/new.
- [ ] If you have a macOS system and can replicate the problem on macOS with Homebrew, please report the issue to [Homebrew/core](https://github.com/Homebrew/homebrew-core) and follow their issue template from your macOS system. If you are unsure, please report the issue to Linuxbrew.
- [x] Ran `brew update` and retried your prior step?
- [x] Ran `brew doctor`, fixed all issues and retried your prior step?
- [ ] Ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [ ] If `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

```
 brew install ipython@5
```
gives the following error:

```
Already downloaded: /scratch/.cache/ipython@5--ipykernel-4.6.1.tar.gz
==> python -c import setuptools... --no-user-cfg install --prefix=/scratch/.linuxbrew/Cellar/ipython@5/5.5.0_2/libexec/vendor --single-version-externally-managed --record=installed.txt
==> /scratch/.linuxbrew/Cellar/ipython@5/5.5.0_2/libexec/bin/ipython kernel install --prefix /scratch/tmp/d20180124-23867-j1ll05
Error: inreplace failed
/scratch/.linuxbrew/Cellar/ipython@5/5.5.0_2/share/jupyter/kernels/python2/kernel.json:
  expected replacement of "\"argv\": [\n  \"/usr/bin/python\"," with "\"argv\": [\n  \"/scratch/.linuxbrew/opt/python/bin/python2\","
```

**Comment by iMichka** [comment]
Hi. Yes, I saw that here: #5661

I had no time to investigate further. This inreplace also exists on mac, so I am wondering why this is failing now. 

I have more urgent matters to fix for the moment, as brew 1.5.0 and 1.5.1 need be merged; so I will not have time to look at this before a few days.

**Comment by tjohns** [comment]
I'm experiencing this as well.

(Bump to make sure this issue doesn't get auto-closed.)
